azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -174,6 +174,11 @@ func resourceArmFunctionApp() *schema.Resource {
 				Default:  false,
 			},
 
+			"daily_memory_time_quota": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
```

Comment:
looks like if the value of `DailyMemoryTimeQuota` is set to 0 or a negative value, then the quota will be turned off. I know `schema.TypeInt` will default to a value of 0 if not provided. Do we want to explicitly set a default value here?

Reply:
I think that should be fine - since the quota itself only affects functions inside a consumption plan?